sources/web/datalab/idleTimeout.ts

```diff
@@ -45,10 +45,25 @@ export function init() {
   }
   if (shutdownCommand) {
     let idleTimeoutStr = userSettings.idleTimeoutInterval;
-    logging.getLogger().debug('idleTimeoutStr from user settings: ' + idleTimeoutStr);
+    if (idleTimeoutStr === undefined) {
```

Comment:
Do we need the if? Doesn't the original line already print `undefined` if it's undefined?